### PR TITLE
Fix GET managers and fix PATCH request endpoint

### PR DESCRIPTION
**Pull request by mfrank37**
# Description
As on the mockup: https://www.figma.com/file/hM1DgjHISuEDVenacmEyo7/Barefoot-Nomad?node-id=546%3A0
 assigning users to managers requires to select from manager list options.

Therefore, I added the `id`, `last_name`, `profile_picture`... which will help to identify exactly which manager to be assigned the user(s). Also the id will be needed in making the PATCH request.

I also changed the patch request to use `query string` instead of `body`, as it's recommended.

## Type of change

Please check the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How should this be manually tested or reproduced?

Open the endpoint `/api/v1/assignUserstoManager/verified-users` for a `GET` request on the local development server and add Authorization header of a manager.
You should be able to get a list of `managers` with the following columns.
`id`, `first_name`, `last_name`, `profile_picture`, `email`, `username`, `address` and `role`.

----------------------------------
Then Check on the other endpoint `/api/v1/assignUserstoManager/verified-users/:id` to make a `PATCH` request.
Add a query parameter of `manager_id` with an id of a manager.
example: `/api/v1/assignUserstoManager/verified-users/f2baf776-b866-4c75-8089-fa10bcf0e694?manager_id=fb94de4d-47ff-4079-89e8-b0186c0a3be8`
 You should be able to assign that user to the specified manager.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)

### CODE REVIEW

File: src/controllers/findAllManagers.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 const listOfManager = async function (req, res,next) {
   const getAllManagers = await getManagers(req,res)
   res.status(200).send({
       title: 'Managers',
-      getAllManagers,
-    
-  })
-}
-export default listOfManager 
+      getAllManagers
```
Comment (CYIMANA-Faisal) [inline src/controllers/findAllManagers.js:8]: Why not not giving the names that are meaningful so that the returned data should be easy to access.

**Review by CYIMANA-Faisal — APPROVED** [review]

File: src/controllers/findAllManagers.js (hunk 1 above)
Comment (mfrank37) [reply src/controllers/findAllManagers.js:8]: The names and other data are returned from the function `getManagers(req, res)` found at [here](https://github.com/atlp-rwanda/c4-barefoot-backend/pull/37/files/ddbb180191caee71acc857717015b4dac9f992f8#diff-c1006b8b20059176f2b25f8e256b7a0b617297caee0bf754aadd1f2e80ad8890)
I hope I answered you.

**Review by salviosage — COMMENTED** [review]
Please rebase

**Comment by Alice-Gatera** [comment]
test your action, reducer and component

**Comment by mfrank37** [comment]
@Alice-Gatera , this backend require actions or components.

**Comment by Anguandia** [comment]
@mfrank37 , I do not see how to manually test this fix, what to expect, etc. Please include that in you PR documentation

**Comment by mfrank37** [comment]
@Anguandia , I have updated the PR documentation.

**Review by Noel2002 — APPROVED** [review]

**Review by Alice-Gatera — APPROVED** [review]

**Review by salviosage — APPROVED** [review]
LGTM

**Pull request merged**